hw3.py

```diff
@@ -0,0 +1,32 @@
+#3.
+tries = 1
+import random
+game = ["student", "homework", "vacation", "notebook", "pencil"]
+print("You have to guess the following word related to school.")
+a = random.choice(game)
+correct = a
+length = len(a)
+length = str(length)
+b = input("The word is " + length + " letters long. Guess a letter: ")
```

Comment:
No need for length to be a string (line 8 and 9 can be removed). It can be used "..." + str(len(a)) + ""